src/styles/project.st.css

```diff
@@ -0,0 +1,135 @@
+/**
+ * Shelley Project - project.st.css
+ * 
+ * Our main project which is essentially a theme but it itself can have themes
+ * so 'project' is more appropriate.
+ *
+ * This project is to be used in conjunction with Shelley's 'default' 
+ * styles|theme|project whatever you want to call it.
+ *
+ * Shelley's default provides the base and this sits on top of that and here
+ * we can import and override the CSS variables declared as part of the default. 
+ * We also set all of our project specifics such as colours and fonts. 
+ *
+ * Limitatons: We use CSS vars so IE11 is mostly out the window but fallback can be provided. 
+ * We are also unable to use override the values of the breakpoints and that's about it.
+ *
+ * You are able to clone our default and just use that as a fork and go from 
+ * there, that way you can override everything at the base and use that as your
+ * go to for your projects. Up to you.
+ */
+
+@namespace "Shelley";
+
+/* = Default project vars to be overridden. */
+:import {
+  -st-from: "@actionishope/shelley/styles/default/project.st.css";
+  -st-named:
```

Comment:
These are CSS runtime variables that can be imported from our base project file and then overridden.

Take a look here to see what you can override, you need to look at the -- vars, the ones above that are build time vars which are currently not overridable but most of them are mapped to CSS vars here possible.

https://github.com/action-is-hope/shelley/blob/master/src/styles/default/project.st.css